#横向控制相关参数
k_pure: 0.75 # PurePursuit"增益系数
k_cte: 5.0  #Stanley"增益系数


#LQR Q R矩阵参数
Q_ed: 10.0
Q_ed_dot: 1.0
Q_ephi: 15.0
Q_ephi_dot: 1.0
R_value: 20.0

#纵向控制PID
kp: 0.5
ki: 0.05
kd: 0.05

